src/program/apis/__init__.py

```diff
@@ -7,18 +7,19 @@
 from .overseerr_api import OverseerrAPI, OverseerrAPIError
 from .plex_api import PlexAPI, PlexAPIError
 from .trakt_api import TraktAPI, TraktAPIError
-
+from program.apis.tvmaze_api import TVMazeAPI
```

Comment:
`from .tvmaze_api import TVMazeAPI`

Also be sure to make a custom exception for TVMaze as well. Follow the same concepts as the other api modules do in this api's dir